File: DSA/Searching/Linear/C/linear.c
#include<stdio.h>
int linear(int arr[],int len,int target){
    for (int i = 0; i < len; i++)
    {
        if(arr[i]==target){
            return i;
        }
    }
    return -1;
    
}
int main(){
    int num;
    printf("Enter the Length: ");
    scanf("%d",&num);
    int arr[num];
    for (int i = 0; i < num; i++)
    {
        scanf("%d",&arr[i]);
    }
    int target;
    printf("Enter the Target: ");
    scanf("%d",&target);
    int ans=linear(arr,num,target);
    printf("%d",ans);

    return 0;
    
}

File: DSA/Searching/Linear/Python/linear.py
def linear(num,target):
    ln=len(num)
    for i in range(ln):
        if num[i]==target:
            return i

if __name__=="__main__":
    num=list(map(int,input("Enter the Elements :").split()))
    target=int(input("Enter the target: "))
    ans=linear(num,target)
    print(ans)
        